lib/dor/text_extraction/ocr.rb

```diff
@@ -85,7 +85,7 @@ def cleanup_input_folder
 
         files = Dir.glob("#{abbyy_input_path}/*")
         logger.info "Removing ABBYY input directory: #{abbyy_input_path}.  Empty: #{Dir.empty?(abbyy_input_path)}. Num files/folders: #{files.count}: #{files.join(', ')}"
-        FileUtils.rm_r(abbyy_input_path)
+        delete_folder(abbyy_input_path)
```

Comment:
I wonder if `FileUtils.rm_rf` would work here too.

Reply:
It actually is an alias and does the same thing with the same problem: https://ruby-doc.org/stdlib-2.4.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-rm_rf

Adding the `force` param simply ignores the (incorrect) exception and ends up not deleting the empty folder and then proceeding (already tried that).